[Serve] Rolling update (#3249)
skypilot-org/skypilot: docs/source/running-jobs/environment-variables.rst

  * - ``SKYPILOT_SETUP_NODE_IPS``
    - A string of IP addresses of the nodes in the cluster with the same order as the node ranks, where each line contains one IP address.
    - 1.2.3.4

>   * - ``SKYPILOT_SERVE_REPLICA_ID``

Reviewer: ```suggestion
  * - ``SKY_SERVE_REPLICA_ID``
```
to discuss; not sure

Author: Let's keep `SKYPILOT` to be align with the other environment variables.

>     - The ID of a replica within the service (starting from 1). It will only appear when the task is run as a :ref:`service <sky-serve>`.

Reviewer 2: ```suggestion
    - The ID of a replica within the service (starting from 1). Available only for a :ref:`service <sky-serve>`'s replica task.
```
Same below?
